File: CS/WinWebSolution.Module/DemoViewController.cs
using System;
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Actions;

namespace WinWebSolution.Module {
    public class DemoViewController : ViewController {
        public DemoViewController() {
            saDemoAction = new SimpleAction(this, "DemoAction", DevExpress.Persistent.Base.PredefinedCategory.Unspecified);
            saDemoAction.TargetObjectType = typeof(ISupportDemoAction);
        }
        private SimpleAction saDemoAction;
        public SimpleAction DemoAction {
            get { return saDemoAction; }
        }
    }
}

File: VB/WinWebSolution.Module/DemoViewController.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Actions

Namespace WinWebSolution.Module
	Public Class DemoViewController
		Inherits ViewController
		Public Sub New()
			saDemoAction = New SimpleAction(Me, "DemoAction", DevExpress.Persistent.Base.PredefinedCategory.Unspecified)
			saDemoAction.TargetObjectType = GetType(ISupportDemoAction)
		End Sub
		Private saDemoAction As SimpleAction
		Public ReadOnly Property DemoAction() As SimpleAction
			Get
				Return saDemoAction
			End Get
		End Property
	End Class
End Namespace
